### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by GraceHtet**
Implement the design from the sneak peek wireframes
Screenshot 2023-06-15 195032
Screenshot 2023-06-15 195335
Screenshot 2023-06-15 195413
Screenshot 2023-06-15 195348

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+    <% @user.recent_post.each do |post|%>
+        <%= link_to "/users/#{@user.id}/posts/#{post.id}" do%>
+            <%= render partial: 'shared/post_card', object: post,  as: :post%>
+            <div class="comment-card">
+                <%= render partial:'shared/comment_card', collection: post.recent_comment, as: :comment %>
+            </div>
+        <% end%>
+    <% end %>
+    <%= link_to 'See All Posts', "/users/#{@user.id}/posts"%>
+</main>
```
Comment (muneebulrehman) [inline app/views/users/show.html.erb]: As per the wireframe design provided, there should be a post number at the top of each post. Kindly implement the same in your app.

| Your design  | Wireframe design |
| ------------- | ------------- |
| image | image |

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+                    <%= render partial: 'shared/post_card', locals: {post: post} %>
+                    <div class="comment-card">
+                        <%= render partial:'shared/comment_card', collection: post.recent_comment, as: :comment %>
+                    </div>
+                <% end  %>
+            </div>
+        <%end%>
+    <% end%>
+
+</main>
```
Comment (muneebulrehman) [inline app/views/posts/index.html.erb:22]: Same here. Kindly display the post number along with the post content just like in the wireframe.

| Your design  | Wireframe design |
| ------------- | ------------- |
| image | image |

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @GraceHtet , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Great job using partial views
- Great job using the `recent_posts` and `recent_comments` methods to fetch the most recent data.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @GraceHtet, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

 

## Highlights
- No linter errors ✔️ 
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**